api/v1beta1/nodemaintenance_types.go

```diff
@@ -61,6 +61,12 @@ type NodeMaintenanceStatus struct {
 	// Phase is the represtation of the maintenance progress (Running,Succeeded,Failed)
 	//+operator-sdk:csv:customresourcedefinitions:type=status
 	Phase MaintenancePhase `json:"phase,omitempty"`
+	// The drain's elapsed time
+	//+operator-sdk:csv:customresourcedefinitions:type=status
+	ElapsedTime float64 `json:"elapsedTime,omitempty"`
+	// The drain's progress in percentage
+	//+operator-sdk:csv:customresourcedefinitions:type=status
+	DrainProgress float64 `json:"drainProgress,omitempty"`
```

Comment:
Originally, I was looking for a double like in other program language and I came up with float as the closest match.
I used _float64_ for the type since I though it is a little bit more meaningful than an integer number (string is optional as well).
I added it as well in the PR's description.